api/src/main/java/org/datacleaner/api/ShortNews.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.api;
+
+import java.util.Date;
+import java.util.List;
+
+public class ShortNews implements java.io.Serializable {
+
+    List<Item> newsItems;
+
+    public ShortNews() {
+    }
+
+    public ShortNews(List<Item> newsItems) {
+        this.newsItems = newsItems;
+    }
+
+    public List<Item> getNewsItems() {
+        return newsItems;
+    }
+
+    public void setNewsItems(List<Item> newsItems) {
+        this.newsItems = newsItems;
+    }
+
+    public static class Item implements java.io.Serializable {
+
+        private String name;
+        private String title;
+        private Date dateCreated;
+        private String message;
+
+        public Item() {
```

Comment:
We discussed this a bit offline, and while it's fixable, it's probably worse than just keeping it mutable.